sonic-chassisd/scripts/chassisd

```diff
@@ -630,6 +913,61 @@ class ChassisdDaemon(daemon_base.DaemonBase):
         else:
             self.log_warning("Caught unhandled signal '{}' - ignoring...".format(SIGNALS_TO_NAMES_DICT[sig]))
 
+    def set_initial_dpu_admin_state(self):
+        """Send admin_state trigger once to modules those are powered up"""
+        lock = threading.Lock()
+        threads = []
+        for module_index in range(0, self.module_updater.num_modules):
+            op = None
+            # Get midplane state of DPU
+            module_info_dict = self.module_updater._get_module_info(module_index)
+            module_name = platform_chassis.get_module(module_index).get_name()
+            state_db = daemon_base.db_connect("STATE_DB")
+            midplane_table = swsscommon.Table(state_db, CHASSIS_MIDPLANE_INFO_TABLE)
+            fvs = midplane_table.get(module_name)
+            midplane_state = 'False'
+            if isinstance(fvs, list) and fvs[0] is True:
+                fvs = dict(fvs[-1])
+                midplane_state = fvs[CHASSIS_MIDPLANE_INFO_ACCESS_FIELD]
+
+            try:
+                if module_info_dict is not None:
+                    # Get admin state of DPU
+                    key = module_info_dict[CHASSIS_MODULE_INFO_NAME_FIELD]
+                    admin_state = self.module_updater.get_module_admin_status(key)
```

Comment:
As per [this comment](https://github.com/sonic-net/sonic-platform-daemons/pull/467/files#r1796139055) and [this comment](https://github.com/sonic-net/sonic-platform-daemons/pull/467/files#r1796155286) the `set_initial_dpu_admin_state` is supposed to handle the dark mode requirement (If there is no CONFIG_DB `CHASSIS_MODULE` entries for the DPU then the admin state for the DPU should be down) but in this function, the `get_module_admin_status` [would return `'up'`](https://github.com/sonic-net/sonic-platform-daemons/blob/2ae658dbc20fe7d8862fd0319e1cf78151aec105/sonic-chassisd/scripts/chassisd#L357) by default if there is no CONFIG_DB entry, meaning `op = 'up'` which would power on the dpu, but by default the DPU should be powered off by default if there is no CONFIG_DB entry